scripts/react/server-safe-globals.js

```diff
@@ -68,12 +68,6 @@ const globalThisShim = {
   },
   CustomEvent: function CustomEvent() {},
   getComputedStyle: function () {},
-  navigator: {
-    languages: [],
-    get userAgent() {
-      return '';
-    },
-  },
```

Comment:
`globalThis.navigator` was added to Node in v21.0.0, which breaks the `react/build.js` script because its can't be altered. Moved to conditional below.